src/Tracy/Bar/panels/info.panel.phtml

```diff
@@ -32,6 +32,13 @@ $ipFormatter = static function (?string $ip): ?string {
 	return $ip;
 };
 
+if(Debugger::$behindProxy === true) {
```

Comment:
Please fix codestyle: `if (Debugger::$behindProxy === true) {` (space between `if` and bracket).

Reply:
thx